modules/installation-osp-deploying-bare-metal-machines.adoc

```diff
@@ -30,6 +29,8 @@ Be sure that your `install-config.yaml` file reflects whether the {rh-openstack}
 
 * Bare metal is available as link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/bare_metal_provisioning/configuring-the-bare-metal-provisioning-service-after-deployment#creating-the-bare-metal-flavor_bare-metal-post-deployment[a {rh-openstack} flavor].
 
+* Bare metal workers do not function on clusters that run on {rh-openstack} versions between 16.1.6 and 16.2.4 due to a link:https://bugzilla.redhat.com/show_bug.cgi?id=2033953[known issue] that causes the metadata service to be unavailable for services on {product-title} nodes.

```

Comment:
Suggestion: Is it possible to start with a sentence that provides an explicit condition for users to meet, such as "Your {rh-openstack} version is either less than 16.1.6 or later than 16.2.4" and then follow with this added text for context? 

Either way I think users would get the message to avoid those problematic versions, but being more explicit might be a nice touch.

Reply:
```suggestion
* If your cluster runs on an {rh-openstack} version that is more than 16.1.6 and less than 16.2.4, bare metal workers do not function due to a link:https://bugzilla.redhat.com/show_bug.cgi?id=2033953[known issue] that causes the metadata service to be unavailable for services on {product-title} nodes.

```